pvlib/iotools/psm3.py

```diff
@@ -63,7 +63,7 @@
 
 def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
              attributes=ATTRIBUTES, leap_day=None, full_name=PVLIB_PYTHON,
-             affiliation=PVLIB_PYTHON, url=None, map_variables=None,
+             affiliation=PVLIB_PYTHON, map_variables=None, url=None,
```

Comment:
Note for posterity: `url` is new in `get_psm3` in this version (#1736), so this isn't a change relative to 0.9.5